src/store/modules/pollList.js

```diff
@@ -0,0 +1,83 @@
+import Axios from "axios";
+
+export default {
+  state: {
+    pollList: [],
+    id: "",
+    title: "",
+    options: "",
+    response: {},
+    responseError: "",
+    progress: false,
+    error: false,
+    isModalActive: false
+  },
+  getters: {
+    poll: state => state.pollList,
+    progressbar: state => (state.progress ? true : false)
+  },
+  actions: {
+
+    async showPollList({ commit }, payload) {
+      const response = await Axios.get("http://192.168.1.7:8000/list_polls", {
+        headers: {
+          api_token: payload.token
+        }
+      }).then(response => (this.info = response));
+      commit("list", response);
+      this.state.response = true;
+    },
+
+    async delete({ dispatch, commit }, payload) {
+      try {
+        commit("progress", true);
+        await Axios.delete(
+          `http://192.168.1.7:8000/delete_poll/${payload.pollid}`,
```

Comment:
this url should come from environment file. not hard code like this. nothing should be hardcoded